framework/src/ble_manager/ble_manager_state.c

```diff
@@ -197,6 +267,137 @@ ble_result_e blemgr_handle_request(blemgr_msg_s *msg)
 		ret = ble_drv_conn_is_any_active(is_active);
 	} break;
 
+	// Scanner
+	case BLE_EVT_CMD_START_SCAN: {
+		BLE_STATE_CHECK;
+
+		if (g_scan_ctx.state != BLE_SCAN_STOPPED) {
+			ret = TRBLE_BUSY;
+			break;
+		}
+
+		ble_scan_state_e priv_state = g_scan_ctx.state;
+		blemgr_msg_params *param = (blemgr_msg_params *)msg->param;
```

Comment:
I think null state of msg & msg->param should be checked at the start of this function.

Reply:
OK. I will fix it